pages/products/products.css

```diff
@@ -0,0 +1,33 @@
+.sidebar-container{
+    position: fixed;
+    left: 0%;
+    margin-top: 2.5rem;
+    background-color: var(--lynx-white-color);
+    height: 100vh;
+    width: 17rem;
+    padding: 3rem 1.5rem;
+    overflow-y: auto;
+}
+
+.category,.rating-label,.sort-label{
+    padding: 5px 0;
+}
+
+.container-box{
+    margin-bottom: 1.5rem;
+}
+
+.price-range{
+    width: 100%;
+}
+
+.product-container{
+ padding: 7rem 20rem;

```

Comment:
try to avoid give padding to much

Reply:
Done [b1873e9](https://github.com/Manpreet-hub/ecommerceApp/pull/2/commits/b1873e9a732973b0f376de3fd3dcc783eb01191b)